UPGRADE.md

```diff
@@ -8,6 +8,20 @@ awareness about deprecated code.
 
 # Upgrade to 4.0
 
+## Disallowed partial version numbers in ``serverVersion``
+
+The ``serverVersion`` connection parameter must consist of 3 numbers:
+
+```diff
+-'serverVersion' => 8.0
++'serverVersion' => 8.0.0
```

Comment:
We should encourage people to use the actual server version here. Oracle is not really following SemVer. And they're shipping features in 8.0.x releases nowadays.

```suggestion
+'serverVersion' => 8.0.31
```